### PR TITLE
feat: add docs deployment from ape console

### PR DESCRIPTION
**Pull request by Ninjagod1251**
### What I did

fixes: #

This is will improve #651.
I want to add a note link users to Ape Academy for an in depth guide on how to deploy with ape console. I kept it brief.

Please let me know if it clear and if this is a good idea to link here.

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: docs/userguides/transactions.md [hunk 1]
```diff
@@ -20,6 +20,20 @@ def deploy():
     account = accounts.load("MyAccount")
     return account.deploy(project.MyContract)
 ```
+
+### Deployment from ape console to a test network
+
+Deploying from `ape console` allows you to interact with a contract in real time. 
+
+```bash
```
Comment (unparalleled-js) [inline docs/userguides/transactions.md:28]: The code block says `bash` but contains both `python` and `bash`

File: docs/userguides/transactions.md [hunk 2]
```diff
@@ -20,6 +20,20 @@ def deploy():
     account = accounts.load("MyAccount")
     return account.deploy(project.MyContract)
 ```
+
+### Deployment from ape console to a test network
```
Comment (unparalleled-js) [inline docs/userguides/transactions.md:24]: This seems really specific. could we just say `"Deployment"` ?

File: docs/userguides/transactions.md [hunk 3]
```diff
@@ -20,6 +20,20 @@ def deploy():
...
+
+Deploying from `ape console` allows you to interact with a contract in real time. 
+
+```bash
+ape console --network :goreli:alchmey
```
Comment (unparalleled-js) [inline docs/userguides/transactions.md:29]: Both `goerli` and `alchemy` are spelled incorrectly here

File: docs/userguides/transactions.md [hunk 4]
```diff
@@ -20,6 +20,20 @@ def deploy():
...
+dev = accounts.load("dev")
+dev.deploy(project.Token) 
+token.contract_method_defined_in_contract()
+```
+
+For an in depth tutorial on how to deploy please visit [ApeAcademy](https://apeworx-academy.webflow.io/)
```
Comment (unparalleled-js) [inline docs/userguides/transactions.md:35]:
```suggestion
For an in depth tutorial on how to deploy, please visit [ApeAcademy](https://apeworx-academy.webflow.io/).
```

File: docs/userguides/transactions.md [hunk 5]
```diff
@@ -20,6 +20,20 @@ def deploy():
...
+
+### Deployment from ape console to a test network
+
+Deploying from `ape console` allows you to interact with a contract in real time. 
+
+```bash
+ape console --network :goreli:alchmey
+dev = accounts.load("dev")
+dev.deploy(project.Token) 
+token.contract_method_defined_in_contract()
```
Comment (unparalleled-js) [inline docs/userguides/transactions.md:32]: `token` is not defined anywhere.

**Review by unparalleled-js — CHANGES_REQUESTED** [review]
This example does not work!

**Review by unparalleled-js — APPROVED** [review]
LGTM

**Review by NotPeopling2day — APPROVED** [review]

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**